Update make-a-plant-watering-robot.md (#3248)
viamrobotics/docs: docs/tutorials/projects/make-a-plant-watering-robot.md

Refer back to this diagram as you complete the steps to wire your hardware.

> <p><img src="/tutorials/plant-watering-pi/full-wiring.png" class="imgzoom" alt="The full wiring diagram for all the hardware for the Plant Watering Robot."></p>

Author: ```suggestion
<p><img src="../../tutorials/plant-watering-pi/full-wiring.png" class="imgzoom" alt="The full wiring diagram for all the hardware for the Plant Watering Robot."></p>
```

> <p><img src="../../tutorials/plant-watering-pi/full-wiring.png" class="imgzoom" alt="The full wiring diagram for all the hardware for the Plant Watering Robot."></p>

Author: ```suggestion
<p><img src="../../plant-watering-pi/full-wiring.png" class="imgzoom" alt="The full wiring diagram for all the hardware for the Plant Watering Robot."></p>
```

> i{{<imgproc src="/tutorials/plant-watering-pi/full-wiring.png" resize="300x" class="imgzoom" declaredimensions=true alt="The full wiring diagram for all the hardware for the Plant Watering Robot.">}

Author: ```suggestion
{{<imgproc src="/tutorials/plant-watering-pi/full-wiring.png" resize="300x" class="imgzoom" declaredimensions=true alt="The full wiring diagram for all the hardware for the Plant Watering Robot.">}}
```

> {{<imgproc src="/tutorials/plant-watering-pi/full-wiring.png" resize="300x" class="imgzoom" declaredimensions=true alt="The full wiring diagram for all the hardware for the Plant Watering Robot.">}}

Author: ```suggestion
<p><img src="/tutorials/plant-watering-pi/full-wiring.png" class="imgzoom" alt="The full wiring diagram for all the hardware for the Plant Watering Robot."></p>
```
